docker-centos/Dockerfile

```diff
@@ -1,12 +1,20 @@
 FROM centos
 
-LABEL maintainer="Giuseppe Scrivano <gscrivan@redhat.com>"
+LABEL maintainer="Giuseppe Scrivano <gscrivan@redhat.com>" \
+      name="docker-centos" \
+      version="0.1" \
+      atomic.type="system" \
+      architecture="x86_64"
 
-RUN yum install -y docker container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin docker-lvm-plugin lvm2 iptables procps-ng xz inotify-tools && mkdir -p /usr/lib/modules && yum-config-manager --nogpgcheck --add-repo https://cbs.centos.org/repos/virt7-docker-el-candidate/x86_64/os/ && yum install -y --nogpgcheck docker-rhel-push-plugin && yum clean all
+RUN yum install --setopt=tsflags=nodocs -y docker container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin docker-lvm-plugin lvm2 iptables procps-ng xz inotify-tools && \
```

Comment:
Might want to have the `rpm -V` step after `yum install` to ensure that all packages were installed. `yum install` will exit as successful even if one of the packages in the list could not be installed.